src/UNIV2LPOracle.t.sol

```diff
@@ -821,7 +829,7 @@ contract UNIV2LPOracleTest is DSTest {
         assertEq(daiEthLPOracle.zzz(), 0);
     }
 
-    function test_stop_start_poke() public {
+    function test_void_start_poke() public {
```

Comment:
maybe also add a `test_stop_void_start_poke()` test? Maybe that is overkill, what do you think?

Reply:
Added!

Reply:
Resolving after discussion, no harm with the additional test.